queries/conditional-tokens/queries.ts

```diff
@@ -124,6 +125,16 @@ const getConditionQuery = gql`
   }
 `;
 
+const getUserQuery = gql`
+  query GetCondition($id: ID!) {
```

Comment:
this name seems wrong

```suggestion
  query GetUser($id: ID!) {
```